### DISCUSSION TITLE
Audio 1 • 20220409: Enabling CamillaDSP makes system unusable

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#878] [Fixed Issues]
Just for fun I enabled CamillaDSP after update to 20220409. After that the mpd.conf contains:
```
audio_output {
        name           "(no sound device)"
        type           "fifo"
        path           "/tmp/mpd.fifo"
        format         "44100:16:1"
}
```
Restoring the last backup is no remedy.

**Comment by rern** [comment]
Please wait for new image files.
There are packages to be installed and configured.

Almost there.

**Comment by klassikmann** [comment]
No hectic pace.I'm up and running again. To enable CamillaDSP was an act of curiosity only.

EDIT: real given name deleted

----- Nachricht beantworten -----
Von: "rern" ***@***.***>
An: "rern/rAudio-1" ***@***.***>
Cc: "klassikmann" ***@***.***>, "Author" ***@***.***>
Betreff: [rern/rAudio-1] Audio 1 • 20220409: Enabling CamillaDSP makes system unusable (Discussion #878)
Datum: Sa., Apr. 9, 2022 18:50

Please wait for new image files.

There are packages to be installed and configured.
Almost there.

&mdash;Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by klassikmann** [comment]
Fixed with rAudio 1 • 20220410
